modules/sdk-coin-apt/src/lib/transactionBuilder/transactionBuilder.ts

```diff
@@ -63,10 +63,22 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
+  /**
+   * @deprecated - use `recipients()`.
+   */
   recipient(recipient: Recipient): this {
     this.validateAddress({ address: recipient.address });
     this.validateValue(new BigNumber(recipient.amount));
-    this.transaction.recipient = recipient;
+    this.transaction.recipients[0] = recipient;
```

Comment:
This might end up in runtime error, what do you think of this suggestion.

```suggestion
    this.transaction.recipients = [recipient];
```

Reply:
@at31416 I was about to give this error, but then had a thought about how the test cases is not getting runtime exception. Then I asked chatgpt and turns out that it is allowed in typescript.